### DISCUSSION TITLE
Unknown table 'COLUMN_STATISTICS' in information_schema (1109)

### DISCUSSION THREAD

**Discussion by ShapesGraphicStudio** [spatie/laravel-backup#1233] [Q&A]
Hi,

`php artisan backup:run` 

gives me error: 

`The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Couldn't execute 'SELECT COLUMN_NAME, JSON_EXTRACT(HISTOGRAM, '$."number-of-buckets-specified"') FROM information_schema.COLUMN_STATISTICS WHERE SCHEMA_NAME = 'database_name' AND TABLE_NAME = 'failed_jobs';': Unknown table 'COLUMN_STATISTICS' in information_schema (1109)`

Is there a way to turn statistics off in config file?

Best regards,
David

**Comment by j5k** [comment]
I have the same issue. My solution is from [here](https://serverfault.com/questions/912162/mysqldump-throws-unknown-table-column-statistics-in-information-schema-1109):

`mysqldump --column-statistics=0`

or

```
[mysqldump]
column-statistics=0
```
to a MySQL config file, such as /etc/my.cnf or ~/.my.cnf.

**Comment by micc83** [comment]
For those who will come, as an alternative, you can instruct **laravel-backup** to use the flag `--column-statistics=0` based on some condition, changing the  `config/database.php` file as follow:
```php
'mysql' => [
      'driver' => 'mysql',
      'host' => env('DB_HOST', '127.0.0.1'),
      '...' => '...',
      'dump' => env('APP_ENV') === 'local' ? ['doNotUseColumnStatistics'] : []
],
```
You can check for all available settings [here](https://github.com/spatie/db-dumper/blob/main/src/Databases/MySql.php).
